Find a functionally equivalent code:android.media.MediaRouter.RouteInfo.isConnecting:COMMENT
Method Modifier: public      
Comment:/**
 * Returns true if the route is in the process of connecting and is not
 * yet ready for use.
 *
 * @return True if this route is in the process of connecting.
 */

@@ -1,3 +1,3 @@
 {
-    return mStatusCode == STATUS_CONNECTING;
+    return mResolvedStatusCode == STATUS_CONNECTING;
 }

